envi/symstore/resolver.py

```diff
@@ -217,6 +217,9 @@ def delSymbol(self, sym):
             if sym.fname in self.symobjsbyname:
                 self.symobjsbyname.pop(sym.fname, None)
 
+            if symval in self.symobjsbyaddr:
+                self.symobjsbyaddr.pop(symval, None)
+
```

Comment:
remove cached address-> mappings during delSymbol.

in the new system, this lookup should track all symbols which point to a given address.